core/src/test/resources/com/google/googlejavaformat/java/testdata/B20128760.output

```diff
@@ -89,19 +89,13 @@ class B20128760 {
   {
     Stream<ItemKey> itemIdsStream =
         stream(members)
-            .flatMap(
-                m ->
-                    m.getFieldValues()
+            .flatMap(m -> m.getFieldValues()
```

Comment:
That's fair enough.

The reason I suggested this change is because, when I last used Ruby about 3 years ago, I seem to remember that the preferred formatting style of lambda expressions, when used with Ruby's functional equivalent of for-loops, follow this style. And I just happened to remember that style recently, and thought to myself that it introduces a sort of visual break that makes things a bit easier to scan IMO, compared to your current proposal of having more on the same line but consequently more tightly-packed and IMO more difficult to read.

I hope that this clarifies my viewpoint, and that perhaps it convinces you and your colleagues to consider this option. :)